src/modules/NtpClient.rb

```diff
@@ -35,8 +35,22 @@ class NtpClientClass < Module
 
     NTP_FILE = "/etc/chrony.conf".freeze
 
-    # The cron file name for the synchronization.
-    CRON_FILE = "/etc/cron.d/suse-ntp_synchronize".freeze
+    TIMER_FILE = "yast-timesync.timer"
+    # The file name of systemd timer for the synchronization.
+    TIMER_PATH = "/etc/systemd/system/#{TIMER_FILE}".freeze
+
+    # Timer content which should be passed to format and seconds added.
+    # @example usage
+    # format(TIMER_CONTENT, timeout: 15)
+    TIMER_CONTENT = <<~TIMER
+      [Timer]
```

Comment:
I usually do not like heredoc strings as they make editing less convenient, e.g. the editors usually do not support syntax highlighting, syntax checks, etc...

I'd prefer to put it into a separate template file, e.g. `yast-timesync.timer.erb` which would contain the ERB template. You can render it like [here](https://github.com/yast/yast-registration/blob/9d6e06ead9f5fb784ab38bb34b9995ca72cd946d/src/lib/registration/helpers.rb#L165) in yast2-registration.

This would also avoid the duplication of the content in the *.spec file, you could simply call

    /usr/bin/erb timeout=<timeout> /.../yast-timesync.timer.erb > /etc/systemd/system/yast-timesync.timer

instead of the copy&paste string in the *.spec file. This would ensure that the content is the same, we do not need to fix two places. That's fragile.